pkg/cmd/cli/describe/printer.go

```diff
@@ -161,6 +161,21 @@ func PrintTemplateParameters(params []templateapi.Parameter, output io.Writer) e
 	return nil
 }
 
+// Receives a resource kind and its name, returns a formatted
+// string with the "resource_kind/resource_name"
```

Comment:
The comment is outdated, and should follow the convention of `formatResourceName (...).`